lib/cli.js

```diff
@@ -425,6 +446,30 @@ const cli = {
             return 0;
         }
 
+        if (options.inspectConfig) {
+
+            // we can only use this option with flat config
+            if (!usingFlatConfig) {
+                log.error("The --inspect-config option is only available when using an eslint.config.js file.");
```

Comment:
the actual result is:
```bash
Invalid option '--inspect-config' - perhaps you meant '--inline-config'?
```

seems the code was never excuted?

Reply:
Oops, good call. Forgotten during refactoring.